src/app/services/auth.service.ts

```diff
@@ -69,9 +69,9 @@ export class AuthService {
           photoURL: '',
           provider: userInfo.providerData[0].providerId
         };
-        this.userService.info.subscribe( user => {
-          this.userInfo = user;
-          if (this.userInfo === null){
+        this.userService.info.subscribe( userDetails => {
```

Comment:
unsubscribe this & all the other subscription on this page

Reply:
@Dhruvkumarshah  Please review my changes